tests/Services/SearchServiceTest.php

```diff
@@ -150,8 +150,11 @@ public function testHandleSearchQuery()
 
         $itemSearchServiceMock = $this->getMockBuilder(ItemSearchService::class)
             ->disableOriginalConstructor()
-            ->setMethods([])
             ->getMock();
+        $itemSearchServiceMock->expects($this->once())
+            ->method('getResult')
+            ->willReturn(['success' => false, 'total' => 0]);
```

Comment:
Returning 'success' shouldn't be necessary anymore, plenty doesn't provide this anymore in the responses of the new search factories.

Reply:
Why do we check it in the code then?
https://github.com/findologic/plugin-plentymarkets-ceres-api/blob/0674c6a6d9b2d3d9817b232fd42ca9a9d82eac40/src/Services/SearchService.php#L354

Reply:
This is an old version, please check develop: https://github.com/findologic/plugin-plentymarkets-ceres-api/blob/develop/src/Services/SearchService.php#L359

Reply:
Why is the `ssr_update_deps` branch not updated with develop changes?

Reply:
That's where I'm currently on: https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/315
Ready to be reviewed soon.

Reply:
If I would change this here now, the tests would fail. So I'll stick with it, until the code was also updated.